internal/api/registry/replication_test.go

```diff
@@ -322,10 +322,21 @@ func TestReplicationForbidAnonymousReplicationFromExternal(t *testing.T) {
 			if err != nil {
 				t.Fatal(err.Error())
 			}
+			// enable anonymous pull and replication on test2/bar
+			err = s2.DB.Insert(&keppel.RBACPolicy{
+				AccountName:             "test1",
+				RepositoryPattern:       "bar",
+				CanPullAnonymously:      true,
+				CanFirstPullAnonymously: true,
+			})
+			if err != nil {
+				t.Fatal(err.Error())
+			}
```

Comment:
Why are we setting an RBAC policy on an irrelevant repository here?

Reply:
On second thought this should be tested in the policy_test.